dataactcore/migrations/versions/19c18c1545c3_file_status_job_unique.py

```diff
@@ -1,4 +1,4 @@
-"""file_status_job_unique
+"""file_job_unique
```

Comment:
I'm assuming nothing breaks with this no longer matching the file name?  I'll test alembic migrations later to make sure.